### PR TITLE
Update interactivetool_metashrimps.xml and associated Docker image

### PR DESCRIPTION
**Pull request by yvanlebras**
(Please replace this header with a description of your pull request. Please include *BOTH* what you did and why you made the changes. The "why" may simply be citing a relevant Galaxy issue.)
(If fixing a bug, please add any relevant error or traceback)
(For UI components, it is recommended to include screenshots or screencasts)

## How to test the changes?
(Select all options that apply)
- [ ] I've included appropriate [automated tests](https://docs.galaxyproject.org/en/latest/dev/writing_tests.html).
- [ ] This is a refactoring of components with existing test coverage.
- [ ] Instructions for manual testing are as follows:
  1. [add testing steps and prerequisites here if you didn't write automated tests covering all your changes]

## License
- [x] I agree to license these and all my past contributions to the core galaxy codebase under the [MIT license](https://opensource.org/licenses/MIT).

### CODE REVIEW

File: tools/interactive/interactivetool_metashrimps.xml [hunk 1]
```diff
@@ -10,7 +10,7 @@
         </entry_point>
     </entry_points>
     <command><![CDATA[
-
+        sed -i 's?input\$file\$datapath?"'$inputfile'"?g' /srv/shiny-server/sample-apps/META/App_metadig/app.R &&
```
Comment (bgruening) [inline tools/interactive/interactivetool_metashrimps.xml:13]: Oh gosh, what is that :)

Comment (bgruening) [reply tools/interactive/interactivetool_metashrimps.xml:13]: are you sure `$file` is not treaded as cheetah variable?

Comment (bgruening) [reply tools/interactive/interactivetool_metashrimps.xml:13]: can you ship your own app.R?

Comment (yvanlebras) [reply tools/interactive/interactivetool_metashrimps.xml:13]: > are you sure $file is not treaded as cheetah variable?

It is workin gproperly on my local planemo instance, so apparently, not ;)

Comment (bgruening) [reply tools/interactive/interactivetool_metashrimps.xml:13]: Ok, we can try ... :)

Comment (yvanlebras) [reply tools/interactive/interactivetool_metashrimps.xml:13]: > can you ship your own app.R?

Duno the best manner to do, and it seems to me having something like that, "just" modifying the existing app source codes with a ""transparent"" function is better than making copy / pasting of source code. You don't think so ? You prefer I propose to replace the app.R ?

**Pull request merged**